blankwindow.py

```diff
@@ -9,33 +9,40 @@
 clock=pygame.time.Clock() #to fix framerate
 
 #importing images 
-ship_surf = pygame.image.load('hit-get/graphics/ship.png').convert_alpha()
+#importing ship
+ship_surf = pygame.image.load('./graphics/ship.png').convert_alpha()
+#importing background
+bg_surf=pygame.image.load('./graphics/background.png').convert_alpha()
 
-#x is constant as player moves slowly to the top.
-ship_y_pos=500  
+#movements
+#movement of ship
+ship_rect=ship_surf.get_rect(center=(window_width/2,window_height/2))  
 
-#importing background
-bg_surf=pygame.image.load('hit-get/graphics/background.png').convert_alpha()
```

Comment:
Why did not you use convert_alpha() for importing background?

Reply:
The reason is- convert simply reduces png to a gameable thing. With alpha, it makes the image background transparent. We need not use alpha to make the background image transparent.